showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITServerSideStreaming.java

```diff
@@ -46,29 +41,11 @@ public class ITServerSideStreaming {
   private EchoClient httpjsonClient;
 
   @Before
-  public void createClients() throws IOException, GeneralSecurityException {
+  public void createClients() throws Exception {
```

Comment:
I think the general recommendation is to keep the exceptions as specific as possible. In terms of documenting, it makes it easier for readers to understand what the method throws/why.
(https://effectivejava.googleplex.com/third-edition#page=325).

Reply:
I completely agree with the recommendation for methods in non-test classes. However, for test methods, especially in this case is a `setup` method, we don't need to understand which methods throws what checked exception, all we care is to have a client created. So it could be a distraction to have a lot of specific checked exceptions, by declaring a generic `Exception` could help testers to focus on the scenario as well.

Of course this could be subjective, there are some good [discussions](https://stackoverflow.com/questions/2496525/is-it-against-best-practice-to-throw-exception-on-most-junit-tests?rq=3) on StackOverFlow as well.